Source/SpatialGDK/Private/SpatialNetDriver.cpp

```diff
@@ -169,6 +169,44 @@ bool USpatialNetDriver::IsLevelInitializedForActor(const AActor* InActor, const
 	return true;
 }
 
+
+void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessTravel /*= false*/)
+{
+	// Remove the actor from the property tracker map
+	RepChangedPropertyTrackerMap.Remove(ThisActor);
+
+	FActorDestructionInfo* DestructionInfo = NULL;
+	const bool bIsServer = ServerConnection == NULL;
+
+	if (bIsServer)
+	{
+		for (int32 i = ClientConnections.Num() - 1; i >= 0; i--)
+		{
+			UNetConnection* Connection = ClientConnections[i];
+			if (ThisActor->bNetTemporary)
```

Comment:
Nit: `{`

Reply:
Just to make sure: this is copied from the UnrealEngine - but when overriding we should probably apply our coding guidelines?

Reply:
I think it depends. If you're doing a block copy of code, it should be bracketed in comment blocked and remain identical. You've removed some functionality, so matching our coding styles seems logical.